src/vm/memory/segments.zig

```diff
@@ -219,6 +220,15 @@ pub const MemorySegmentManager = struct {
     }
 };
 
+// Utility function to help set up memory segments
+//
+// # Arguments
+// - `segment_manager` - MemorySegmentManger to be passed in
+// - `vals` - complile time structure with heterogenous types
+pub fn segmentsUtil(segment_manager: *MemorySegmentManager, comptime vals: anytype) !void {
+    try memoryFile.memoryInner(segment_manager.memory, vals);
```

Comment:
there was a PR with renaming `memoryInner` to `setUpMemory` you will need to change this line